WebAPI/Controllers/TimetableController.cs

```diff
@@ -44,10 +44,13 @@ public async Task<IActionResult> SetStudentTimetableFromGroup([FromBody] Student
 
             Student student = user.Student;
             if (student == null)
-            {
-                student = new Student();
+            {
```

Comment:
It is big logic for controller here in my opinion. It should be implemented in service somewhere...
user contain student (it is not good) it should be oposite...